### ISSUE TITLE
Flyway ignore checksum for repeatable migration file - R__

### ISSUE THREAD

**Issue by mavencode01** [flyway/flyway#2032]
###### Which version and edition of Flyway are you using?
5.0.7

###### If this is not the latest version, can you reproduce the issue with the latest one as well?
(Many bugs are fixed in newer releases and upgrading will often resolve the issue)

###### Which client are you using? (Command-line, Java API, Maven plugin, Gradle plugin)
Maven plguin

###### Which database are you using (type & version)?
Postgres

###### Which operating system are you using?
Unix

###### What did you do?
(Please include the content causing the issue, any relevant configuration settings, the SQL statement that failed (if relevant) and the command you ran.)

I have a repeatable migration script file  (R_tables.sql) that is expected to run each time and
the script suppose to create a new table based on the placeholder arguments: 
`-Dflyway.placeholders.lf_version=<new_version>`

sample script:

```

CREATE TABLE reports_${lf_version}
(
    id SERIAL PRIMARY KEY NOT NULL,
    ...
}    
```

###### What did you expect to see?
Re-run the REPEATABLE migration script (*R_...*) regardless whether the `checksum` change or not.
A repeatable script should not use the checksum and runs the entire script.

###### What did you see instead?
The repeatable migration runs only once successfully and does not re-run the migration to
create the new tables or whatever the content of a REPEATABLE script does.

**Comment by axelfontaine** [comment]
Duplicate of #1453
